recipe/meta.yaml

```diff
@@ -7,22 +7,25 @@ package:
 source:
   fn: cloud_sptheme-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/c/cloud_sptheme/cloud_sptheme-{{ version }}.tar.gz
-  sha256: e55448c14087a7fcbba7d58d6d9edf12190b73ea9c1ca68562ca75aeabbe484a
+  sha256: caf47ac4c6346eef47fc11e799adbeaf645921c712cc099cf2530560e7ecde44
+  patches:
+    - 0001-patch-sphinx-4-support.patch
 
 build:
   noarch: python
-  number: 0
-  preserve_egg_dir: True
+  number: 1
+  preserve_egg_dir: true
   script: {{ PYTHON }} -m pip install . -vv
 
+
 requirements:
   host:
     - pip
-    - python
+    - python >=3.6
```

Comment:
It doesn't appear that they have actually dropped Python 2. Could we please drop this constraint?